coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CoralToTrinoSqlCallConverter.java

```diff
@@ -74,15 +74,15 @@ protected SqlCall transform(SqlCall sqlCall) {
             null),
         // string functions
```

Comment:
Could you add the comment clarifying the reason why we need to use `greatest` here?

Reply:
not using greatest anymore